server/methods/core/orders.js

```diff
@@ -843,11 +843,9 @@ export const methods = {
             });
 
             // Temporarily(?) put here the Shippo's method/label purchasing.After a succesfull capture fund
-            if (order.shipping[0].shipmentMethod.shippoMethod) {
+            if (order.shipping[0].shipmentMethod.settings) {
```

Comment:
A hook implementation( "eg onPaymentCapture")  maybe could be a nice improvement